data/fro/Stroński_1906_EliasDeBarjolsVI/stemma.gv

```diff
@@ -0,0 +1,16 @@
+digraph {
+archetype [color="grey", label=" "] ;
+x [color="grey"] ;
+y [color="grey"] ;
+z [color="grey"] ;
+Kd [label="K (d)"] ;
+archetype -> x ;
+archetype -> y ;
+archetype -> z ;
+x -> L ;
+y -> C ;
+y -> R ;
+z -> G ;
+z -> Kd ;

```

Comment:
Is that two different manuscripts ?

Reply:
Given the conventions, I think that _d_ is the hypothetical model of K (and I), so that would be something like:

```gv
z -> d;
d -> K;
d[color="grey"];
```